src/System.Net.HttpListener/src/System/Net/Windows/WebSockets/HttpWebSocket.Windows.cs

```diff
@@ -252,60 +203,6 @@ internal static void ValidateInnerStream(Stream innerStream)
             }
         }
 
-
-        internal static void ValidateOptions(string subProtocol,
-            int receiveBufferSize,
-            int sendBufferSize,
-            TimeSpan keepAliveInterval)
-        {
-            // We allow the subProtocol to be null. Validate if it is not null.
-            if (subProtocol != null)
-            {
-                WebSocketValidate.ValidateSubprotocol(subProtocol);
-            }
-
-            ValidateBufferSizes(receiveBufferSize, sendBufferSize);
-
-            if (keepAliveInterval < Timeout.InfiniteTimeSpan) // -1
-            {
-                throw new ArgumentOutOfRangeException(nameof(keepAliveInterval), keepAliveInterval,
-                    SR.Format(SR.net_WebSockets_ArgumentOutOfRange_TooSmall, Timeout.InfiniteTimeSpan.ToString()));
-            }
-        }
-
-        internal static void ValidateBufferSizes(int receiveBufferSize, int sendBufferSize)
-        {
-            if (receiveBufferSize < WebSocketBuffer.MinReceiveBufferSize)
```

Comment:
I removed them here in the second commit: https://github.com/dotnet/corefx/pull/19761/commits/f7078b76e92c23576d735e5dab8ab5f2f5c45083#diff-3f8f0d1e604c4e521e7c1f7dc83cd197L28